packages/components/src/DateField/DateField.tsx

```diff
@@ -1,152 +1,71 @@
-import { DateValue, createCalendar } from '@internationalized/date';
-import { HTMLAttributes, ReactElement, RefObject, useRef } from 'react';
+import { ReactElement, forwardRef } from 'react';
+import type RAC from 'react-aria-components';
+import { DateField, DateValue } from 'react-aria-components';
 
-import { AriaDateFieldProps, useDateField } from '@react-aria/datepicker';
-import { useFocusRing } from '@react-aria/focus';
-import { useLocale } from '@react-aria/i18n';
-import { useHover } from '@react-aria/interactions';
-import { mergeProps } from '@react-aria/utils';
+import { WidthProp } from '@marigold/system';
 
-import { useDateFieldState } from '@react-stately/datepicker';
+import { FieldBaseProps } from '../FieldBase';
```

Comment:
Here are the wrong Props used or? Should be from _Fieldbase

Can we get rid of the old Fieldbase?

Reply:
After this one maybe :)

Reply:
ups... should be noticed by the DateField migration :D resolved